dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+      dontrun: TRUE
+      code: |
+            # Warning: The <multicol> tag is obsolete, it might not work as intended.
+            # Try to avoid using it.
+    - name: htmlNav
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlNav(
+                  list(
+                    htmlA("Plotly", href = "https://plot.ly/"),
+                    '> ",
+                    htmlA("Dash", href = "https://plot.ly/dash"),
+                    '> ",
+                    htmlA("Request Trial", href = "https://go.plot.ly/dash-enterprise-trial")
+                  )
+                )
+              ))
+            )
+
+            app$run_server()
+    - name: htmlNextid
+      dontrun: TRUE
+      code: |
+            # Warning: The <nextid> tag is obsolete, as of HTML Version 3.2.
+
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlA(href = "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nextid",
+                      "Click here for more information about this tag")
+              ))
+            )
+
+            app$run_server()
+    - name: htmlNobr
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlNobr("Lorem ipsum dolor sit amet,
+                         consectetur adipiscing elit, sed do eiusmod
+                         tempor incididunt ut labore et dolore magna aliqua.
+                         Ut enim ad minim veniam, quis nostrud exercitation
+                         ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                         Duis aute irure dolor in reprehenderit in voluptate
+                         velit esse cillum dolore eu fugiat nulla pariatur.
+                         Excepteur sint occaecat cupidatat non proident,
+                         sunt in culpa qui officia deserunt mollit anim id est laborum."
+                )
+              ))
+            )
+
+            app$run_server()
+    - name: htmlNoscript
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlH2("Warning: A browser without support for JavaScript
+                      will show the text inside the noscript element.
+                      Since Dash uses JavaScript internally,
+                      using this option might cause incompatibility issues."),
```

Comment:
I doubt it... probably just wouldn't show up at all. Anyway since it would need js in order to create it, the only reasonable option is to put `<noscript>` in the index template.

Reply:
I didn't write this example, so I'm not sure. But that makes sense to me.

Reply:
fixed in 46a624c